src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -227,6 +224,79 @@ private static bool TryGetTextForOperator(SyntaxToken token, Document document,
+                        text = Keyword("optionalParameter");
+                        return true;
+                    }
+                    else if (token.Parent.Parent.IsKind(SyntaxKind.PropertyDeclaration))
+                    {
+                        text = Keyword("propertyInitializer");
+                        return true;
+                    }
+                    else if (token.Parent.Parent.IsKind(SyntaxKind.EnumMemberDeclaration))
+                    {
+                        text = Keyword("enum");
+                        return true;
+                    }
+                    else if (token.Parent.Parent.IsKind(SyntaxKind.VariableDeclarator))
+                    {
+                        text = Keyword("=");
+                        return true;
+                    }
+                }
+                else if (token.Parent.IsKind(SyntaxKind.NameEquals))
+                {
+                    if (token.Parent.Parent.IsKind(SyntaxKind.AnonymousObjectMemberDeclarator))
+                    {
+                        text = Keyword("anonymousObject");
+                        return true;
+                    }
+                    else if (token.Parent.Parent.IsKind(SyntaxKind.UsingDirective))
+                    {
+                        text = Keyword("using");
+                        return true;
+                    }
+                    else if (token.Parent.Parent.IsKind(SyntaxKind.AttributeArgument))
+                    {
+                        text = Keyword("attributeNamedArgument");
+                        return true;
+                    }
+                }
+                else if (token.Parent.IsKind(SyntaxKind.LetClause))
+                {
+                    text = Keyword("let");
+                    return true;
+                }
+                else if (token.Parent is XmlAttributeSyntax)
+                {
+                    // redirects to https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags
+                    text = "see";
+                    return true;
+                }
+
+                // EqualsToken in assignment expression is handled by syntaxFacts.IsOperator call above.
+                // Here we try to handle other contexts of EqualsToken.
+                // If we hit this assert, there is a context of the EqualsToken that's not handled.
+                // In this case, we currently fallback to https://docs.microsoft.com/dotnet/csharp/language-reference/operators/assignment-operator
+                Debug.Fail("Falling back to F1 keyword for assignment token.");
+                text = Keyword("=");
+                return true;
+            }
+
+            if (token.IsKind(SyntaxKind.LessThanToken, SyntaxKind.GreaterThanToken))
+            {
+                if (token.Parent.IsKind(SyntaxKind.FunctionPointerParameterList))
+                {
+                    // PROTOTYPE: Figure out where to redirect.
+
```

Comment:
@BillWagner @333fred Is there a documentation page we can redirect to here?

Reply:
Not yet. I'll ping you and @RikkiGibson on the docs issue referenced below when I open that PR.

Reply:
@BillWagner Sorry for not clarifying. This was about function pointers

https://github.com/dotnet/roslyn/blob/64c5fd150f87ddea7a5fa673555ea2ad74a75e58/src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs#L291-L295

I found now that https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/unsafe-code is the most suitable page.

Opened https://github.com/dotnet/docs/pull/28669.